packages/jsii-reflect/package.json

```diff
@@ -28,6 +28,7 @@
     "jest": "^24.1.0",
     "jsii-build-tools": "^0.8.2",
     "jsii-calc": "^0.8.2",
+    "jsii": "^0.8.2",
     "ts-jest": "^24.0.0",
     "typescript": "^3.3.3333"
```

Comment:
If you depend on `jsii`, you shouldn't depend on `typescript` anymore...

Reply:
Strongly disagree. Just because dependencies exist somewhere down the dependency tree, doesn't mean I should just invisibly use them. You should always declare your direct dependencies, it's the only way to do proper dependency management.

If we were to rewrite `jsii` to Java, `jsii-reflect` would break for no good reason.

If we were to need to depend on a different version of `typescript` between these 2 packages, `jsii-reflect` would break for no good reason.

Reply:
Why do you depend on `jsii` then? If it's not used instead of `tic`, then I don't think it should be depended on at all...

Reply:
We depend on `jsii` because the unit tests depend on jsii.

Reply:
I am with Rico.